com.unity.shadergraph/Documentation~/View-Direction-Node.md

```diff
@@ -4,6 +4,8 @@
 
 Provides access to the mesh vertex or fragment's **View Direction** vector. This is the vector from the vertex or fragment to the camera. The coordinate space of the output value can be selected with the **Space** dropdown parameter.
 
+NOTE: In versions prior to 11.0, the **View Direction** vector was not normalized in the **Universal Render Pipeline**. Version 11.0 changed this behavior, and this vector is now normalized in both the **High-Definition Render Pipeline** and the **Universal Render Pipeline**. To mimic old behavior, you can use the [Position Node](Position-Node.md) in **World** space and subtract the **Position** output of the [Camera Node](Camera-Node.md). 
```

Comment:
Do you know if the camera position output in (local) World space in HDRP?  (i.e. 0,0,0) ?

Reply:
Yeah, that was something i wan going to have to double check in HDRP. I think you are right tho, and will address it

Reply:
This wording is getting changed tho, since weve added the new node